projects/step-frontend/src/lib/modules/timeseries/modules/chart/injectables/tooltip-placement-function.ts

```diff
@@ -58,9 +58,10 @@ export class TooltipPlacementFunction {
       screenBoundaries[secondaryAttributes.after] - screenBoundaries[secondaryAttributes.before] - totalMargin;
     const overlayMaxSize = parseInt(overlayStyle[secondaryAttributes.maxSize]);
 
-    if (!overlayMaxSize || availableSpace < overlayMaxSize) {
-      overlay.style[secondaryAttributes.maxSize] = `${availableSpace}px`;
-    }
+    // uncomment this if max height/width are needed on the tooltip element.
+    // if (!overlayMaxSize || availableSpace < overlayMaxSize) {
+    // overlay.style[secondaryAttributes.maxSize] = `${availableSpace}px`;
+    // }
```

Comment:
I don't understand, is max height or width needed? If not we can remove this

Reply:
@neogucky The tooltip used to have a generated max height and width. It was creating issues on the new custom tooltips. I don't know if this won't affect any other tooltip in some specific situations (at least for now). Since this part is not obvious how to put it back, I preffered to leave it like this. The entire file will be removed after we have a stable https://exense.atlassian.net/browse/SED-3650
I will delete these for now